warning: key "Encoding" in group "Desktop Entry" is deprecated
error: value "stardict.png" for key "Icon" in group "Desktop Entry" is an icon name with an extension, but there should be no extension
error: value "Dictionary" in key "Categories" in group "Desktop Entry" requires another category to be present among the following categories: Office;TextTools

--- dict/data/stardict.desktop.in
+++ dict/data/stardict.desktop.in
@@ -1,11 +1,10 @@
 [Desktop Entry]
-Encoding=UTF-8
 _Name=StarDict
 _Comment=Lookup words
 Exec=stardict
 Terminal=false
 Type=Application
-Icon=stardict.png
+Icon=stardict
 StartupNotify=true
-Categories=Utility;Dictionary;
+Categories=Utility;Office;TextTools;Dictionary;
 X-GNOME-DocPath=stardict/stardict.xml
